app/components/primer/avatar_stack_component.rb

```diff
@@ -10,6 +10,9 @@ class AvatarStackComponent < Primer::Component
 
     DEFAULT_TAG = :div
     TAG_OPTIONS = [DEFAULT_TAG, :span].freeze
+
+    DEFAULT_BODY_TAG = :div
+    BODY_TAG_OPTIONS = TAG_OPTIONS = [DEFAULT_TAG, :span].freeze
```

Comment:
Should we mention this in the docs?

Reply:
Yeah, I think we should. `tag` isn't explicitly part of the arguments and I'm a little blocked by the constraints of how we generate documentation. I'll have to revisit this with fresh eyes on Monday :)